scripts/prebuild.js

```diff
@@ -40,6 +41,8 @@ function prebuildify () {
       `--devdir=${cache}`,
       '--release',
       '--build_v8_with_gn=false',
+      '--v8_enable_pointer_compression=""',
+      '--v8_enable_31bit_smis_on_64bit_arch=""',
```

Comment:
What are these for?

Reply:
Newer flags that are required or it won't compile. Using `""` should use the default value (either true or false).